editor/editor_help.cpp

```diff
@@ -3818,7 +3827,20 @@ void EditorHelpBitTooltip::show_tooltip(EditorHelpBit *p_help_bit, Control *p_ta
 	tooltip->add_child(p_help_bit);
 	p_target->get_viewport()->add_child(tooltip);
 	p_help_bit->update_content_height();
+	// When FLAG_POPUP is false, it prevents the editor from losing focus when displaying the tooltip.
+	// This way, clicks and double-clicks are still available outside the tooltip.
+	tooltip->set_flag(Window::FLAG_POPUP, false);
 	tooltip->popup_under_cursor();
+	_current_tooltip = tooltip;
+}
+
+bool EditorHelpBitTooltip::is_tooltip_visible() {
+	return _current_tooltip;
+}
+
+Control *EditorHelpBitTooltip::get_invisible_control() {
+	ERR_FAIL_NULL_V(_current_tooltip, nullptr);
+	return _current_tooltip->_invisible_control;
 }
```

Comment:
```suggestion
Control *EditorHelpBitTooltip::make_invisible_control() {
	Control *control = memnew(Control);
	control->set_visible(false);
	return control;
}
```

There is no point in storing `_invisible_control` in `EditorHelpBitTooltip`. The control returned by `make_custom_tooltip()` will be automatically freed when the standard tooltip disappears.

Also `_current_tooltip` is no longer needed.

Reply:
I refactored the method `EditorHelpBitTooltip::show_tooltip` to received the symbol text and an optional description. This was, I could put all the display logic in one place. I still need a static variable to prevent the custom tooltip to be displayed multiple times.